content/archive/external/ready4r-netlify-app-labbook.md

```diff
@@ -1,12 +0,0 @@
----
-title: "Ready for R: Notebook Reference"
-author: "Aaron Coyner and Ted Laderas"
-date: "2020-06-12"
-tags: [Package, Course, Data Science]
-link: "https://ready4r.netlify.app/labbook/"
-length_weight: "22.3%"
-repo: "laderast/ready_for_r_site"
-pinned: false
----
-
-This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook. [...] This course introduces you R by working through common tasks in data science: importing, manipulating, and visualizing data. R is a statistical and programming computer language widely used for a variety of applications. Before proceeding with these training materials, please ensure you have an RStudio.cloud account and can see the workspace. This is a searchable website that serves as a reference for the Ready for R course. This gitbook is not meant to be a ...
```

Comment:
@yihui the new description filter ignore some external books that has not changed there description. 

I could change the test to only apply on bookdown.org book, but it is also a great way to detect and maybe warn the writer ?

Reply:
How about testing if the description also contains the sentence "This is a sample book written in Markdown"?